tools/generate_flamegraph.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import argparse
+import subprocess
+import sys
+
+
+def main():
```

Comment:
Is it worth having this file at all? Can we just pass all the args to `py-spy record` and call it a day? It already has a default value with sample_rate so all we really save here is the `-o profile.svg` which I'm not sure is worth this extra file.